runelite-client/src/main/java/net/runelite/client/plugins/camera/CameraConfig.java

```diff
@@ -158,4 +159,27 @@ default boolean compassLook()
 	{
 		return true;
 	}
+
+	@ConfigItem(
+		keyName = "compassMultipleClickCycle",
+		name = "Multiple clicks on compass ",
+		description = "Cycle between Look North, South, East, and West by clicking on compass in rapid succession",
+		position = 11
+	)
+	default boolean compassMultipleClickCycle()
+	{
+		return false;
+	}
+
+	@ConfigItem(
+		keyName = "compassMultipleClickTimeout",
+		name = "Multiple click timeout",
+		description = "Configure how quickly the compass resets to \"Look North\" after the last click",
+		position = 12
+	)
+	@Units(Units.SECONDS)
+	default int compassMultipleClickTimeout()
```

Comment:
This currently isn't used. There's a `RESET_DIRECTION_TIMEOUT` constant which should have its usages replaced by this.

Reply:
My bad, you're right. Fixed.